release/BUILD.bazel

```diff
@@ -384,6 +384,22 @@ py_test(
 # RELEASE TEST INFRA unit tests
 ####
 
+sh_binary(
+    name = "run",
+    srcs = ["run_release_test.sh"],
+    data = ["run_release_test"],
+    env = {
+        "NO_INSTALL": "1",
```

Comment:
we set this to 1 because we don't want to install python dependencies when running with bazel